zadanie2.c

```diff
@@ -0,0 +1,34 @@
+#include <stdio.h>
+
+int main() {
+int a;
+int b;
+int c;
+scanf("%d",&a);
+scanf("%d",&b);
+scanf("%d",&c);
+double outcome1 = (a*b) - ((double)b/(double)c);
+int outcome2 = (a*b) - (b%c);
+int outcome3 = (a*b) - (c*a) - (b*c);
+printf("%f\n",outcome1);
+printf("%d\n",outcome2);
+printf("%d\n",outcome3);
+
+if ((a>b) && (b>c) && (a>c)) {
```

Comment:
jezeli **a** jest wieksze od **b** oraz **b** jest wieksze od **c**, to wiadomo że **a** jest wieksze od **c**.
Ostatni warunek nie jest potrzebny.